### DISCUSSION TITLE
What is the prefered setup to backup different user home dirs on one host running windows 10?

### DISCUSSION THREAD

**Discussion by sjjh** [garethgeorge/backrest#623] [Q&A] [answered]
I've installed and configured backrest for user *bob*, to backup their home dir, using the windows installer as user *bob*. I'd now like to add a backup for user *alice*. Can I simply install and setup backrest under the user account of *alice*, or will two running backrest instances on one host cause problems? The installation seems to happen per user account (instead of system wide) and thus not being visible to another user, but on the other hand the `localhost:port` combo seems *not* to be user specific.
(I tried to install backrest a second time as user *alice* already, which seemed to have caused issues for the installation of user *bob*, e.g., not being able to access the repository and backups anymore... Thus wondering, what best practice is, maybe worth to add to the documentation as well.)

**Comment by homandr** [comment, marked as answer]
Your observations are exactly correct. You can have many Backrest instances under different user accounts as long as each one uses a unique port for Web UI. In your example, add a user variable for _alice_ as described here: https://github.com/garethgeorge/backrest#running-on-windows. Then re-run Backrest installer (make sure to use the latest version) under _alice_ to have the shortcuts updated.

Not sure what version you were installing, but the latest installer will detect another program listening on port 9898 and refuse the installation, also displaying steps needed to address it. This was done precisely to avoid issues like you experienced where a subsequent installation effectively hijacks the Web UI from the previous one.

There is a discussion around running Backrest as a service in #648 although it might not necessarily address your scenario. Maybe you have two or more non-admin users, each wanting to back up their own data. A service would not help with that, the approach above is the proper way to deal with such situation.

**Comment by sjjh** [comment]
FYI, I was using backrest 1.6.2 (lates release at that time) (sorry for not mentioning it).
